tools/test/test_actions_local_runner.py

```diff
@@ -50,11 +52,57 @@ async def test_runner(self) -> None:
                 }
                 f = io.StringIO()
                 with contextlib.redirect_stdout(f):
-                    await actions_local_runner.run_steps([fake_step], "test", None)
+                    await actions_local_runner.run_steps([fake_step], "test", None, True)
 
                 result = f.getvalue()
                 self.assertIn("say hello", result)
                 self.assertIn("hi", result)
 
+        class TestQuicklint(unittest.IsolatedAsyncioTestCase):
```

Comment:
question: do either of these suites actually get run in CI (Test tools / test)? from this log it doesn't look like they do: https://github.com/pytorch/pytorch/runs/2529412059

if that's the case, could you fix this file so that all these tests do get run in CI?